lighthouse-core/audits/dobetterweb/no-console-time.js

```diff
@@ -1,82 +0,0 @@
-/**
- * @license
- * Copyright 2016 Google Inc. All rights reserved.
- *
```

Comment:
@ebidel 

They were the only audits left that had perf impact and would need their own pass. Somewhere there was also a thread with @brendankenny on the dangers of letting these run during the perf pass which is also why I recommended not enabling them until we proved they had minimal impact. 

From the PR description:

> Deletes no-console-time and no-datenow since they monkeypatch functions with capture callsites
> Fancier stuff that got removed could potentially be enabled during the perf pass depending on how heavy the callsite capturing is, but further testing is needed